wpilibNewCommands/src/main/native/cpp/frc2/command/CommandScheduler.cpp

```diff
@@ -93,6 +94,16 @@ CommandScheduler& CommandScheduler::GetInstance() {
   return scheduler;
 }
 
+void CommandScheduler::ResetInstance() {
+  if (!frc::RobotBase::IsSimulation()) {
+    FRC_ReportWarning(
+        "Ignoring call to frc2::CommandScheduler::ResetInstance() outside of "
+        "simulation! frc2::CommandScheduler::ResetInstance() should only be "
+        "used for unit test setup.");
+  }
+  std::make_unique<Impl>().swap(GetInstance().m_impl);
```

Comment:
The if statement should `return` like Java; currently the reset will still happen

Reply:
Ah, thanks for noticing that! I'll hopefully be able to fix that in 6-8 hours.